src/portal/azext_portal/generated/commands.py

```diff
@@ -0,0 +1,22 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import CliCommandType
+
+
+def load_command_table(self, _):
```

Comment:
Add experimental tag for the command group `logic`, you can reference [here](https://github.com/arrownj/azure-cli-extensions/blob/a9c5b01192ba687f6465a7f9a96bc25005f9c339/src/stream-analytics/azext_stream_analytics/commands.py#L16-L17).